src/calendar-app/calendar/gui/EditCalendarDialog.ts

```diff
@@ -144,9 +146,13 @@ export function showCreateEditCalendarDialog({
 		return null
 	}
 
+	const isExternalCalendarCreation = isNewCalendar && isExternalCalendarType(calendarType)
 	Dialog.showActionDialog({
-		title: () => lang.get(titleTextId),
+		title: lang.get(titleTextId),
 		allowOkWithReturn: true,
+		confirmButtonAtTheBottom: isExternalCalendarCreation,
+		titleInsideDialogContent: isExternalCalendarCreation,
+		errorMessageStream: errorMessageStream,
```

Comment:
You don't need new properties. You can:
- give an empty title and insert the title inside the ```.flex.col``` child;
- same for ```confirmButtonAtTheBottom```, just give an empty ```okAction```;

Then you don't need the ```errorMessageStream``` as a parameter also